### DISCUSSION TITLE
Retirement of Visual Studio for Mac effect

### DISCUSSION THREAD

**Discussion by adrianhall** [Azure/azure-mobile-apps#741] [General]
In line with the retirement of Visual Studio for Mac, I will be removing instructions for Visual Studio for Mac from the documentation.  Officially, I can only support Visual Studio now.

Feel free to drop comments, etc. below so I can guage whether this is appropriate and what to do if it's not.

**Comment by RichardFromAltSource** [comment]
Depending on demand, and the currently available content related to Visual Studio for Mac, what do you think about moving it all to a "No Longer Supported" section of the site for a few months to make it easier to track down for folks that still depend on the existing information.  Hate to see information just go away without a phase-out period, even if you no longer respond to questions about it or support it...

**Reply by adrianhall** [reply]
I was thinking of removing it once the retirement of VS Mac becomes effective, so it will be around until August of next year.

**Reply by RichardFromAltSource** [reply]
Seems reasonable.  Thanks for soliciting feedback...
